elasticsearch/example/src/test/java/example/springdata/elasticsearch/conference/ElasticsearchOperationsTest.java

```diff
@@ -103,4 +109,29 @@ void geoSpatialSearch() {
 
 		assertThat(result).hasSize(2);
 	}
-}
+
+    @Test
+    void shouldUpdateConferenceKeywords() {
+        var conferenceName = "JDD14 - Cracow";
+        var newKeyword = "java-ee";
+
+        var initialQuery = new CriteriaQuery(new Criteria("name").is(conferenceName));
+        SearchHit<Conference> searchHit = template.searchOne(initialQuery, Conference.class);
+        assertThat(searchHit).isNotNull();
```

Comment:
Asserting an intermediate state quickly creates the impression that we're testing something that shouldn't be tested. When setting up a state, we assume it is correct and expect the given state, otherwise, the entire arrangement is just waste.